pilot/pkg/networking/util/util_test.go

```diff
@@ -416,59 +416,6 @@ func TestAddSubsetToMetadata(t *testing.T) {
 	}
 }
 
-func TestCloneCluster(t *testing.T) {
```

Comment:
does this result in loss of coverage?

Reply:
CloneCluster function is deleted as a result of it being dead code now